substrate/frame/contracts/src/lib.rs

```diff
@@ -312,10 +326,12 @@ pub mod pallet {
 		///
 		/// Changing this value for an existing chain might need a storage migration.
 		#[pallet::constant]
+		#[pallet::no_default]
 		type DepositPerByte: Get<BalanceOf<Self>>;
```

Comment:
Is a default really not possible here @gupnik ? Thats a bit sad...

Reply:
You an use `pallet::no_default_bounds` here @pgherveou @ggwpez

Reply:
got it and then in TestDefaultConfig I have to hard code the Balance type defined here?
https://github.com/paritytech/polkadot-sdk//blob/0bf6e1ab48f2a3918a95e6001da618a2ef0ed750/substrate/frame/balances/src/lib.rs#L231
So in a way they are all tightly coupled, which I guess is ok for Test config.

How would that work if we wanted to use that for something else than well defined TestConfig?

Reply:
Once you set `no_default_bounds`, internally it just removes all bounds and creates `type DepositPerByte` in the trait DefaultConfig. 

So, you can pretty much set it to anything as long as it makes sense within your TestDefaultConfig.

Reply:
is it worthwhile adding dummy value for these fields though if they are going to be overridden anyway?

Reply:
If you want to ensure that users override this value, its absolutely fine to set `no_default`. Is there no sensible default for `DepositPerByte`?